core/domain/rte_component_registry_test.py

```diff
@@ -67,11 +67,19 @@ def _validate_customization_arg_specs(self, customization_arg_specs):
             self.assertTrue(isinstance(ca_spec['description'], basestring))
             self.assertGreater(len(ca_spec['description']), 0)
 
+            schema_utils_test.validate_schema(ca_spec['schema'])
+            # Default value of SanitizedUrl obj_type may be empty. Empty value
+            # cannot normalized, so we don't normalize it.
+            if ca_spec['schema']['type'] == 'custom' and (
+                    ca_spec['schema']['obj_type'] == 'SanitizedUrl'):
+                if ca_spec['default_value'] == '':
+                    self.assertEqual(len(ca_spec['default_value']), 0)
+                    return
```

Comment:
No, don't return early. There needs to be an actual if/else here, otherwise if we add subsequent checks to the end of this function later, they will not run for this case. (But see my comment on line 78/79.)

Reply:
Done.